numpyro/patch.py

```diff
@@ -1,3 +1,6 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
ditto: add an extra newline

Reply:
Oh I missed this, thanks for catching this. Will update.